uint8 error_code
uint8 error_severity
uint32 error_value